service/access.go

```diff
@@ -347,6 +362,27 @@ func (a *accessService) updateAccessToken(ctx context.Context, domain, role, pro
 	return at.(*AccessTokenResponse), err
 }
 
+func (a *accessService) storeTokenCache(key string, acd *accessCacheData, expTimeDelta time.Time, expTime *jwt.NumericDate) {
+	oldTokenCacheData, _ := a.tokenCache.Get(key)
+	a.tokenCache.SetWithExpire(key, acd, expTime.Sub(expTimeDelta))
+	if oldTokenCacheData != nil {
+		if oldTokenCache, ok := oldTokenCacheData.(*accessCacheData); ok {
+			oldTokenCacheSize := accessCacheMemoryUsage(oldTokenCache)
+			a.memoryUsage += accessCacheMemoryUsage(acd) - oldTokenCacheSize
+		}
+	} else {
+		a.memoryUsage += accessCacheMemoryUsage(acd)
+		a.memoryUsage += int64(len(key))
+	}
```

Comment:
refactor
```suggestion
	if oldTokenCacheData != nil {
		if oldTokenCache, ok := oldTokenCacheData.(*accessCacheData); ok {
			oldTokenCacheSize := accessCacheMemoryUsage(oldTokenCache)
			a.memoryUsage += accessCacheMemoryUsage(acd) - oldTokenCacheSize
			return
		}
		a.memoryUsage += accessCacheMemoryUsage(acd)
		return
	}
	a.memoryUsage += accessCacheMemoryUsage(acd) + int64(len(key))
	return
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/4d64861f9e57b8cd4c3c4bd6b03e6d65fb48dc8a